wallet-new/src/Cardano/Wallet/API.hs

```diff
@@ -54,8 +56,9 @@ type InternalAPI = "api" :> "internal" :> Internal.API
 internalAPI :: Proxy InternalAPI
 internalAPI = Proxy
 
+type WIPAPI = "api" :> "unimplemented" :> WIP.API
```

Comment:
If the path is `unimplemented` - why did you named it `WIPAPI`? Why not `UnimplementedAPI`? :-) Is it too long?

Reply:
To me its sounds strange to name a function `unimplemented` if it is actually at least somehow implemented.
I have no strong opinion about how to name the endpoint itself
Alfredo suggested to use `unimplemented` to make clear that is not meant for end users.